P5/Source/Specs/listBibl.xml

```diff
@@ -34,22 +35,21 @@
   </classes>
   <content>
     <sequence>
-      
-        <classRef key="model.headLike" minOccurs="0" maxOccurs="unbounded"/>
-        <elementRef key="desc" minOccurs="0" maxOccurs="unbounded"/>
-      
-      
-        <alternate minOccurs="1" maxOccurs="unbounded">
-          <classRef key="model.biblLike"/>
-          <classRef key="model.milestoneLike"/>
-        </alternate>
-      
-      
+      <classRef key="model.headLike" minOccurs="0" maxOccurs="unbounded"/>
+      <elementRef key="desc" minOccurs="0" maxOccurs="unbounded"/>
+      <alternate minOccurs="0" maxOccurs="unbounded">
+        <classRef key="model.milestoneLike" minOccurs="1" maxOccurs="1"/>
+        <elementRef key="relation" minOccurs="1" maxOccurs="1"/>
+        <elementRef key="listRelation" minOccurs="1" maxOccurs="1"/>
+      </alternate>
+      <sequence minOccurs="1" maxOccurs="unbounded">
+        <classRef key="model.biblLike" minOccurs="1" maxOccurs="unbounded"/>
         <alternate minOccurs="0" maxOccurs="unbounded">
-          <elementRef key="relation"/>
-          <elementRef key="listRelation"/>
+          <classRef key="model.milestoneLike" minOccurs="1" maxOccurs="1"/>
+          <elementRef key="relation" minOccurs="1" maxOccurs="1"/>
```

Comment:
Just out of curiosity: the addition of `minOccurs="1" maxOccurs="1"` to various `elementRef` is purely cosmetic (for clarity?), right?

Reply:
Yes, that is my recollection. I sometimes think `@minOccurs` and `@maxOccurs` should not have defaults, but rather should be required.